### DISCUSSION TITLE
Multiline grouping

### DISCUSSION THREAD

**Discussion by snowy-owll** [glideapps/glide-data-grid#798] [General]
Is there a way to create a multiline grouping? Something like the picture.
![изображение](https://github.com/glideapps/glide-data-grid/assets/13611053/75eb3928-2a14-445f-963d-b5cc0908894c)

**Comment by jassmith** [comment]
Not currently.

**Reply by snowy-owll** [reply]
Thank you for the reply! Is this expected in future releases? It would be great :)

**Reply by KyDenZ** [reply]
Mulitple cell grouping (row or cols) would be really useful for migrating to this library. Maybe it's possible with TreeView implemented in v6?

**Reply by jassmith** [reply]
We are working on this in 6.x series. It probably will not land for 6.0 but in a minor release after.

**Reply by jnoelvictorino** [reply]
Hi, may I ask if the latest version now supports this feature?
I saw some multiline grouping in the intro picture of the repo.

Thanks!
